graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -447,6 +442,10 @@ def _initialize_metadata_dict(
         metadata_dict["edge_type"] = edge_types
         metadata_dict["node_type"] = sorted(node_type_set)
 
+        # We use the data location as the graph name, can also take from user?
+        # TODO: Fix this, take from config?
```

Comment:
What does the TODO mean?

Reply:
Was planning to include graph name in the config itself, I'll remove this.